.travis.yml

```diff
@@ -14,7 +14,7 @@ install:
   - "pip install -r requirements.txt -r requirements.test.txt"
 before_script:
   - "cd client && npm install && cd .."
-  - "cd client && webpack --config webpack.prod.config.js && cd .."
+  - "cd client && ./node_modules/.bin/webpack --config webpack.prod.config.js && cd .."
```

Comment:
@Empact is this line relevant to this PR?

Reply:
Well it's independent bug it fixes this failure: https://travis-ci.org/embassynetwork/modernomad/builds/280742205?utm_source=github_status&utm_medium=notification

Reply:
I'll separate it out into another PR

Reply:
#297